src/libstore/local-store.cc

```diff
@@ -1856,33 +1847,39 @@ void LocalStore::queryRealisationUncached(const DrvOutput & id,
     }
 }
 
-FixedOutputHash LocalStore::hashCAPath(
-    const FileIngestionMethod & method, const HashType & hashType,
+ContentAddress LocalStore::hashCAPath(
+    const ContentAddressMethod & method, const HashType & hashType,
     const StorePath & path)
 {
     return hashCAPath(method, hashType, Store::toRealPath(path), path.hashPart());
 }
 
-FixedOutputHash LocalStore::hashCAPath(
-    const FileIngestionMethod & method,
+ContentAddress LocalStore::hashCAPath(
+    const ContentAddressMethod & method,
```

Comment:
These I just generalized to not use the old now-deleted type. And this gets us closer in fact to being able to deduplicate this with some `build/local-derivation-goal.cc` stuff later.